src/main/java/io/redlink/more/data/service/GarminService.java

```diff
@@ -168,8 +195,21 @@ private List<ParticipantKeyValue> participantForGarminUserId(String garminUserId
         ).toList();
     }
 
+    private List<ParticipantKeyValue> keyValuesForParticipant(Long studyId, Integer participantId) {
```

Comment:
Da fehlt ein `get` oder `findAll` im Methodennamen ... ist nur eine private Methode, daher ist es nicht schlimm

```suggestion
    private List<ParticipantKeyValue> getKeyValuesForParticipant(Long studyId, Integer participantId) {
```